main.go

```diff
@@ -34,9 +30,9 @@ func checkForErrorsAndExit(err error) {
 	if err == nil {
 		os.Exit(0)
 	} else {
-		logger := util.CreateLogger("")
+		logger := util.CreateLogEntry("", "debug")
 		if os.Getenv("TERRAGRUNT_DEBUG") != "" {
-			logger.Println(errors.PrintErrorWithStackTrace(err))
+			logger.Errorln(errors.PrintErrorWithStackTrace(err))
```

Comment:
If we supported `--terragrunt-log-level`, then we could use `Errorf` on the basic error message (as in the `else` block below) and `Debugf` for the stack trace.

Reply:
@brikis98 the problem here is that `checkForErrorsAndExit()` is in `main` package, and is executed when default logger is not yet available. So changing this will require refactoring in itself - either it is possible to reuse some of `urfave/cli` functionality, or something move this bit to `cli/cli_app`. What do you think?

Reply:
Moving this error handling into `cli` and making logging more consistent seems like a good idea to me!

Reply:
@brikis98 I ended up solving this differently - by introducing a global `LogEntry` object, that gets initiated early. That allows us to log things, and use `--terragrunt-log-level`